console/cli_context.go

```diff
@@ -16,6 +62,101 @@ func (r *CliContext) Arguments() []string {
 	return r.instance.Args().Slice()
 }
 
+func (r *CliContext) Choice(question string, choices []console.Choice, option ...console.ChoiceOption) (string, error) {
+	var answer string
+
+	if len(option) > 0 {
+		answer = option[0].Default
+	}
+
+	options := make([]huh.Option[string], len(choices))
+	for i, choice := range choices {
+		options[i] = huh.NewOption[string](choice.Key, choice.Value).Selected(choice.Selected)
+	}
+
+	input := huh.NewSelect[string]().Title(question).Options(options...)
+	if len(option) > 0 {
+		input.Description(option[0].Description)
+		if option[0].Validate != nil {
+			input.Validate(option[0].Validate)
+		}
+	}
+
+	err := input.Value(&answer).Run()
+	if err != nil {
+		return "", err
+	}
+	return answer, err
+}
+
+func (r *CliContext) Confirm(question string, option ...console.ConfirmOption) (bool, error) {
+	var answer bool
+	if len(option) > 0 {
+		answer = option[0].Default
+	}
+
+	input := huh.NewConfirm().Title(question)
+	if len(option) > 0 {
+		input.Description(option[0].Description).Affirmative(option[0].Affirmative).Negative(option[0].Negative)
+	}
+	err := input.Value(&answer).Run()
+	if err != nil {
+		return false, err
+	}
+
+	return answer, nil
+}
+
+func (r *CliContext) Error(message string) {
+	color.Redln(message)
+}
+
+func (r *CliContext) Info(message string) {
+	color.Blueln(message)
```

Comment:
Modify to Greenln?

Reply:
Info is shown with Blue color but i think we can modify it since laravel uses Green color for it.